app/src/main/java/org/mozilla/fenix/utils/ClipboardHandler.kt

```diff
@@ -19,12 +22,13 @@ class ClipboardHandler(context: Context) {
 
     var text: String?
         get() {
-            if (clipboard.isPrimaryClipEmpty() ||
-                !clipboard.isPrimaryClipPlainText()) {
-                return null
-            }
+            if( !clipboard.isPrimaryClipEmpty() &&
```

Comment:
nit: if statement spacing
```suggestion
            if (!clipboard.isPrimaryClipEmpty() &&
```